### DISCUSSION TITLE
<Link> not working

### DISCUSSION THREAD

**Discussion by pragatheeswarancse** [vercel/next.js#21534] [Help]
**What version of Next.js are you using?**

10.0.1

**What version of Node.js are you using?**

14.15.3

**What browser are you using?**

Chrome

**What operating system are you using?**

windows

**How are you deploying your application?**

Vercel

**Describe the Bug**

Link is not working in my project, it's not show any error in console what can i do
I use Tailwindcss in Next js

**Expected Behavior**

if i click button it has to go Home page but it's not responsible

**To Reproduce**

<div className="pt-3">
                  <Link href="/">
                    <a className="text-white rounded py-1 px-3 bg-indigo-600 hover:bg-indigo-700">Home</a>
                  </Link>
</div>

**Comment by tarunama** [comment]
Did you check tailwindcss example ?
https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss
